### PR TITLE
Remove unnecessary "istio namespace" prom queries in graph generation

### LINKED ISSUE #2891: Remove unnecessary "istio namespace" prom queries in graph generation
In telem V2, and actually I think in later versions of mixer, Istio telemetry no longer generates intra-istio service traffic telemetry.  In graph generation we had special-case support to capture this telemetry when the user selected the istio-system namespace (or whichever ns was configured for the control plane).

We can remove this support, simplify the code, and make the slight perf improvement.

### PR DESCRIPTION
**Pull request by jshaughn**
With istiod (maybe even before given the overhead) Istio no longer reports  telemetry between its components. It also no longer offers component deployment across namespaces. This removes a bunch of special case code dealing with the internal istio time-series.  Note that things like the ingress and egress gateways are unaffected.

Fixes #2891

### CODE REVIEW

File: graph/types.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ const (
 	GraphTypeVersionedApp string = "versionedApp"
 	GraphTypeWorkload     string = "workload"
 	NodeTypeApp           string = "app"
+	NodeTypeOperation     string = "operation" // The special request_operation aggregate node
 	NodeTypeService       string = "service"
```
Comment (jmazzitelli) [inline graph/types.go:17]: This is unrelated to this PR (which should just be about removing the component namespace stuff) so this should be reverted. This "operation" thing is for the upcoming work on the experimental metric classification stuff.

Comment (jshaughn) [reply graph/types.go]: good catch, looks like I had PR spillage... Fixed.

**Comment by jshaughn** [comment]
@mattmahoneyrh This is a code removal, just looking to ensure no graph regressions, with and without the istio namespace selected.
